docs/userguide/configuring.rst

```diff
@@ -1,17 +1,17 @@
 .. _configuration-section:
 
-Configuration
+Environment Configuration
 =============
 
 Parsl workflows are developed completely independently from their execution environment.
 There are very many different execution environments in which Parsl programs and their apps can run, and
-mamy of these enviroments have multiple options of how those programs and apps run, which makes
+mamy of these enviroments have multiple options of how those Parsl programs and apps run, which makes
 configuration somewhat complex, and also makes determining how to set up Parsl's configuration
 for a particular set of choices fairly complex, though we think the actual configuration
 itself is reasonable simple.
 
 Parsl offers an extensible configuration model through which the execution environment and
-communication with that environment is configured. Parsl is configured
+communication within that environment is configured. Parsl is configured
```

Comment:
There are two types of communication that you might be configuring: the channels that determine how you connect to the resource (where communication with makes sense), and the executor itself over which the user doesn't really configure/control the communication. So this with is more correct that within.

Reply:
If the main parsl process is part of the environment we are configuring (which I think it is), then within seems more appropriate to me, since nothing is outside the environment.

Reply:
I can see that, there is the environment as a whole and multiple envs within it.